src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+import java.time.LocalDate;
+
+public class HelloWorld {
+
```

Comment:
why did you remove fields?)

Reply:
I've made a decision that creating and initialization of these fields in two different places are redundant.
But now the method name "initializeVariables" doesn't make sense. (